backend/lib/server_code/api/roomRoutes.js

```diff
@@ -9,20 +11,27 @@ let roomPlayers = [];
 roomRouter.use((req, res, next) => {
   res.date = Date.now();
   console.log("Time: ", Date.now());
+
   next();
 });
 
 roomRouter.post("/create", (req, res) => {
-  const { creatorID } = req.query;
-  roomPlayers.push(creatorID);
+  const { creatorID } = req.body;
+  //roomPlayers.push(creatorID);
```

Comment:
if not needed, remove it

Reply:
I seperate it for 2 actions so you should notice:
1. the user create a new game = new room but he wont join the game untill he press submit to his name
2. after he submit his name - the user enter the room using join in the server.

Reply:
https://github.com/aviadch/Mafia/pull/6/commits/865f308e2aa9d0c7e3beadae889332db95801bc2

Reply:
good job